application-templates-ui/src/main/resources/Templates/Article/Template/WebHome.fr.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.3" reference="Templates.Article.Template.WebHome" locale="fr">
+  <web>Templates.Article.Template</web>
+  <name>WebHome</name>
+  <language>fr</language>
+  <defaultLanguage>en</defaultLanguage>
+  <translation>1</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1554731536000</creationDate>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1554733889000</date>
+  <contentUpdateDate>1554733889000</contentUpdateDate>
+  <version>1.1</version>
+  <title>Modèle d'article</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
```

Comment:
The translation pages should be hidden by default AFAIK. Right now, this cause a regression in our WCAG check because the template contain an inline style.

We don't have the issue with the original template since it's marked as hidden: we skip those pages from our WCAG checks.

Reply:
Well, these are not quite translation pages, but regular pages with translated content. IIRC translation pages are those that hold an object of `XWiki.TranslationDocumentClass` type and that also use syntax `plain/1.0`. At the same time, the rule with hidden pages is about technical pages, which happen to include the translations. In this case your issues might be fixed, but I wonder if, on a FR wiki, you would be able to find these pages with an used that can't find hidden pages.

Reply:
> Well, these are not quite translation pages, but regular pages with translated content.

Well, they are not `translation pages` if I  follow this definition indeed, but they are definitely `translated pages` in the sense that the `translation` property is set to 1.

> In this case your issues might be fixed, but I wonder if, on a FR wiki, you would be able to find these pages with an used that can't find hidden pages.

I don't really understand what might be the issue? The translation pages are already hidden (see: https://github.com/xwiki-contrib/application-templates/blob/master/application-templates-ui/src/main/resources/Templates/Article/Template/WebHome.xml#L41), so this is already true for EN wiki...

Now I try to find back the rules you mentioned and actually I just ended up on https://dev.xwiki.org/xwiki/bin/view/Community/ApplicationDevelopmentBestPractices#HInternationalization and it's indeed not quite clear. I guess we should clarify all that.

FTR I already performed the changes and released the application to hide those pages, I hope it's ok for you.

Reply:
I didn't know that the EN version was hidden. That is something part of the demo content (Help center). When someone starts a fresh wiki, the user can't see hidden pages. So we might have some issue, that, by default, the demo content is hidden. Does this sound good to you?

Reply:
I might miss something, but for me those pages are used to create new pages from templates: they are not supposed to be directly accessed by the users. So it seems consistent to consider them only as technical pages.

Reply:
Page translations should have the same value for the hidden field as the default translation. Either all translations are hidden (including the default one) or all are visible. In this case the Article template is hidden most probably because it is exposed in the Create Page dialog through a template provider. So yes, users are not supposed to access directly the template page, AFAIK.

Reply:
Indeed, there's a template provider that uses that template. All good.